pybamm/parameters/lithium_ion_parameters.py

```diff
@@ -368,31 +376,19 @@ def R_p_dimensional(self, x):
         inputs = {"Through-cell distance (x_p) [m]": x}
         return pybamm.FunctionParameter("Positive particle radius [m]", inputs)
 
-    def a_n_dimensional(self, x):
-        """
-        Negative electrode surface area per unit volume as a function of
-        through-cell distance
-        """
-        inputs = {"Through-cell distance (x_n) [m]": x}
-        if self.options["particle shape"] == "spherical":
-            return 3 * self.epsilon_s_n / self.R_n_dimensional(x)
-        elif self.options["particle shape"] == "user":
-            return pybamm.FunctionParameter(
-                "Negative electrode surface area to volume ratio [m-1]", inputs
-            )
+    def epsilon_s_n(self, x):
```

Comment:
If you set a model for LAM then the variable corresponding to `epsislon_s_n` and the parameter `epsilon_s_n` will be different (I think?) - the latter will just give the initial parameter value/function provided. Will this cause confusion? What you really provide as a parameter is the initial active material volume fraction, but I think it would be messy to append "Initial" everywhere. The same is true for porosity. 

I don't think this needs changing here, but something to think about in the future. I think my opinion is to leave it as is, and that it is understood that if you have a model describing how a parameter changes then that parameter should be treated as a variable.

Reply:
Yes, this is something we should mention when we put together the workflow for degradation modeling. I agree putting "Initial" into the parameter names would be messy